src/license_expression/__init__.py

```diff
@@ -1075,10 +1088,17 @@ def decompose(self):
         yield self.license_symbol
         yield self.exception_symbol
 
-    def render(self, template='{symbol.key}', *args, **kwargs):
+    def render(self, template='{symbol.key}', wrap_with_in_parens=False, *args, **kwargs):
+        """
+        Return a formatted WITH expression. If `wrap_in_parens`, wrap in parens.
```

Comment:
I think `wrap_in_parens` should be `wrap_with_in_parens` to correctly mirror the argument name.

Reply:
I pushed a commit to fix this.